lib/streamlit/testing/element_tree.py

```diff
@@ -134,6 +134,71 @@ def set_value(self, v: Any):
         self._value = v
         return self
 
+    @property
+    def is_button(self) -> Button:
```

Comment:
nit: it could be a bit confusing that it's called `is_button` and returns itself instead of a boolean. Maybe it could be named just `button` which would be a bit similar to how you can call `.str` on a Series in Pandas. 

It also makes it a bit clearer to read, e.g.:

`sr2 = sr.get_widget("cb").is_radio.set_value("on").run()`

vs

`sr2 = sr.get_widget("cb").radio.set_value("on").run()`

Reply:
Calling it `button` would be confusing in a different way, because we already have a `button` method on blocks for getting an iterable of buttons contained within it. `is_button` was the tentative favorite when Josh and I talked about it, but I'll get input from more people before merging.

Reply:
oh, i see. Maybe another alternative might be `as_button`